main/test/ca/uwaterloo/flix/language/feature/Test.Expression.Select.flix

```diff
@@ -0,0 +1,85 @@
+/////////////////////////////////////////////////////////////////////////////
+// Select expressions (channel select)                                     //
+/////////////////////////////////////////////////////////////////////////////
+
+def main(): Int = 42
+
+// Test integers
+def fi01(): Int = select {
+    case x = <- ch => 0
+}
+
+@test
+def testSelectWildcardi01(): Bool = assertEq!(fi01(), 0)
```

Comment:
The tests should be more systematic. In particular, you should test with the following types:

Primitives: Bool, Char, Float32, Float64, Int8, Int16, Int32, Int64, BigInt
Data: Unit, Str, Option[...], Result[...], List[...], and functions (x -> y).